New Topic for Astro Support Request Form (#822)
astronomer/docs: astro/astro-support.md

---
title: 'Submit a Support Request'
navTitle: 'Astro Support'
id: support
description: Get Astro support when you need it.
---

If you're an Astro user and haven't found the answer in Astronomer documentation, the following resources are available:

- [Astronomer Forum](https://forum.astronomer.io)
- [Airflow Guides](https://www.astronomer.io/guides/)

If you're experiencing a more serious issue and need Astronomer expertise, you can use one of the following methods to contact Astronomer Support:

> - Submit a support request in the [Cloud UI](#submit-a-support-request-in-the-cloud-ui)

Reviewer: ```suggestion
- Submit a support request in the [Cloud UI](astro-support.md#submit-a-support-request-in-the-cloud-ui)
```

> - File a support request on the [Astronomer Support Portal](https://support.astronomer.io/hc/en-us)

Reviewer 2: I believe we want users to use the in product support form as their primary means of requesting support. We should state that here.

In the rare cases where they cannot submit a request via the Cloud UI (eg. cannot log in, Cloud UI is down), then use one of the alternatives.

@christine-sherry what do you think?

Author: @chomipi88 this statement appears in the section titled _Submit a Support Request on the Astronomer Support Portal_:

_Astronomer recommends that you submit support requests in the Cloud UI_

Also, having the Cloud Ui appear as the first item in the list of options implies that using the Cloud UI is the preferred method.

Reviewer 3: @chomipi88 I agree, let's call it out as the preferred method in our docs. Thanks guys!

Examples:

- There is a bug in the Software UI.
- Astro CLI usage is impaired (for example, there are incompatibility errors between installed packages).
- There is an Airflow issue that has a code-based solution.
- You received a log alert on Astronomer.

**P4:** Low impact, service fully usable.

Examples:

- You can't find your Workspace.
- There are package incompatibilities caused by a specific, complex use case.
- You have questions about best practices for an action in Airflow or on Astronomer.

### Be as descriptive as possible

The more information you can provide about the issue you're experiencing, the quicker Astronomer Support can start the troubleshooting and resolution process. When submitting a support request, include the following information:

- What project/deployment does this question/issue apply to?
- What did you already try?
- Have you made any recent changes to your Deployment or running DAGs?

### Include logs or code snippets

If you've already copied task logs or Airflow component logs, send them as a part of your request. The more context you can provide, the better.

## Submit a Support Request in the Cloud UI

> 1. In the Cloud UI, click **Help** > **Submit Support Request**.

Reviewer 2: Screenshot of the location in the UI would help guide the user. This is going to remain static.

Reviewer 2: Also may want to add that this can be accessed for all pages in Cloud UI

Author: @chomipi88 I added an image similar to this:

![support-request-location](https://user-images.githubusercontent.com/104205257/174287823-6d615190-7257-4ffb-8b68-b98ba3c57d13.png)

We don't provide images of the entire UI when we need to highlight a specific control. Only the portion of the screen with the control is shown. This is sufficient to help users locate the correct location.

If the control can be accessed anywhere in the UI, there s no need to indicate this.

2. Complete the following fields:

    - **Problem Statement**: Enter a description of the issue you are experiencing. Provide as much detail as possible.
    - **Workspace**: Optional. Select the Workspace where the issue is occurring. This list is auto-populated when you submit a support request from a Workspace or a Deployment.

>     - **Deployment**: Optional. Select the Deployment where the issue is occurring. This list is auto-populated when you submit a support request from a Workspace or a Deployment.

Reviewer 2: Correction: This list is auto-populated when you submit a support request from a Deployment.

Author: @chomipi88 in my experience, Deployments and Workspaces are auto-populated and available within a Workspace _and_ a Deployment.  Happy to demonstrate.

    - **Problem Start**: Optional. Select the date and time the issue started.
    - **Description**: Enter a thorough description of the issue. Provide as much detail as possible. 

>     - **Priority**: Select the severity of your issue. For severity level explanations, see [Best Practices for Submitting Support Requests](#best-practices-for-submitting-support-requests).

Reviewer: ```suggestion
    - **Priority**: Select the severity of your issue. For severity level explanations, see [Best Practices for Submitting Support Requests](astro-support.md#best-practices-for-support-request-submissions).
```

    - **Business Impact**: Optional. Describe how this issue is affecting your organization.
    - **CC Emails**: Optional. Enter the email address of a team member that you want to notify about this issue. Click **Add Additional Email** to add additional email addresses.

3. Click **Submit**.

>     When Astronomer Support start working on your issue, you'll receive email notifications. To check the status of a support request, sign in to the [Astronomer Support Portal](https://support.astronomer.io). 

Reviewer 2: An auto-generated email is sent to the user(s) when the ticket is submitted/created in Zendesk. Future email comms will be sent when Astronomer responds to the ticket.

Author: @chomipi88 the text now reads  _You'll receive an email when your ticket is created and follow-up emails as Astronomer Support processes your request_.

## Submit a Support Request on the Astronomer Support Portal

> Astronomer recommends that you submit support requests in the Cloud UI.

Reviewer: ```suggestion
Astronomer recommends that you submit support requests in the Cloud UI. If you can't access the Cloud UI for any reason, you can also submit report requests through the [Astronomer Support Portal](https://support.astronomer.io).
```
I think we should account for situations where the control plane is down, someone is having auth issues, etc.

Author: @jwitz I used _If you can't access the Cloud UI, you can submit support requests on the [Astronomer Support Portal](https://support.astronomer.io)._

> If you're new to Astronomer, you'll need to create an account on the [Astronomer Support Portal](https://support.astronomer.io) to submit a support request. If you're working with a team and want to view support tickets created by other team members, use your work email or the domain you share with your team members for your account. If your team uses more than one email domain (for example, @astronomer.io), contact Astronomer and ask to have it added manually to your organization.

Reviewer: ```suggestion
If you're new to Astronomer, you'll need to create an account on the Astronomer Support Portal to submit a support request. If you're working with a team and want to view support tickets created by other team members, use your work email or the domain you share with your team members for your account. If your team uses more than one email domain (for example, `@astronomer.io`), contact Astronomer and ask to have the additional domains added to your organization.
```

> If you're an existing customer, sign in to the Astronomer Support Portal and create a new support request.

Reviewer: Are they doing this through the UI? When I open the form submission on the UI, I'm not prompted to log in. Or is this meant to communicate the alternative to using the Cloud UI?

Author: @jwitz you're not prompted to log in because **you already have access**s. This is what new users without access see:

![image](https://user-images.githubusercontent.com/104205257/174343660-9e54ce81-f2c9-4591-b602-5ef890ac6e46.png)

As you can see, if you're not an existing user, "you'll need to create an account".

> If you're an existing customer, sign in to the Astronomer Support Portal and create a new support request.

Reviewer: ```suggestion
If you're an existing customer, sign in to the [Astronomer Support Portal](https://support.astronomer.io) and create a new support request.
```

> ## Monitor Existing Support Requests

Reviewer 2: Support requests submitted via Cloud UI will be managed over email.
Our goal is to have less people trying to register and log into Zendesk as it is a manual and cumbersome process with a poor user experience.

---
title: 'Submit a Support Request'
navTitle: 'Astro Support'

> id: support

Reviewer: ```suggestion
id: astro-support
```

description: Get Astro support when you need it.
---

> If you're an Astro user and haven't been able to resolve an issue using Astronomer documentation, the following resources are available:

Author: @paolaperaza this was updated to: _In addition to product documentation, the following resources are available to help you resolve issues:_

- [Astronomer Forum](https://forum.astronomer.io)
- [Airflow Guides](https://www.astronomer.io/guides/)

> If you're experiencing a more serious issue and need Astronomer expertise, you can use one of the following methods to contact Astronomer Support:

Author: @paolaperaza typically, you want to limit the frequency users contact support. Support calls are expensive and time consuming. Using the term "serious" indicates that users should try resolving an issue on their own first before contacting Astronomer.  As Astronomer grows, you might want to revisit this.

### Be as descriptive as possible

The more information you can provide about the issue you're experiencing, the quicker Astronomer Support can start the troubleshooting and resolution process. When submitting a support request, include the following information:

- What project/deployment does this question/issue apply to?
- What did you already try?
- Have you made any recent changes to your Deployment or running DAGs?

### Include logs or code snippets

If you've already copied task logs or Airflow component logs, send them as a part of your request. The more context you can provide, the better.

## Submit a Support Request in the Cloud UI

1. In the Cloud UI, click **Help** > **Submit Support Request**.

    <div class="text--center">
    <img src="/img/docs/support-request-location.png" alt="Submit Support Request menu location" />
    </div>

2. Complete the following fields:

    - **Problem Statement**: Enter a description of the issue you are experiencing. Provide as much detail as possible.
    - **Workspace**: Optional. Select the Workspace where the issue is occurring. This list is auto-populated when you submit a support request from a Workspace or a Deployment.

>     - **Deployment**: Optional. Select the Deployment where the issue is occurring. This list is auto-populated when you submit a support request from a Workspace or a Deployment.

Author: @paolaperaza I discussed this at length with Piotr [2022-06-17](https://astronomer.slack.com/archives/D03ETDQ6F09/p1655487950378409).

Here is some of the highlights from our conversation:

![image](https://user-images.githubusercontent.com/104205257/174591565-03151661-bd4c-44f7-a105-966560075a7c.png)

![image](https://user-images.githubusercontent.com/104205257/174591759-aa9ae5d4-f9c4-4edc-8baa-bd1322d7e8c5.png)

The current text is accurate. If you want to discuss it further, let me know.

    - **Problem Start**: Optional. Select the date and time the issue started.
    - **Description**: Enter a thorough description of the issue. Provide as much detail as possible. 

>     - **Priority**: Select the severity of your issue. For severity level explanations, see [Best Practices for Submitting Support Requests](astro-support.md#best-practices-for-support-request-submissions).

Author: @paolaperaza I added _Select the severity of your issue based on how the issue is affecting your organization_

    - **Business Impact**: Optional. Describe how this issue is affecting your organization.
    - **CC Emails**: Optional. Enter the email address of a team member that you want to notify about this issue. Click **Add Additional Email** to add additional email addresses.

3. Click **Submit**.

>     You'll receive an email when your ticket is created and follow-up emails as Astronomer Support processes your request. To check the status of a support request, sign in to the [Astronomer Support Portal](https://support.astronomer.io). 

Reviewer 4: ```suggestion
    You'll receive an email when your ticket is created and every time an Astronomer team member replies to your request. To check the status of support requests, you can also sign in to the [Astronomer Support Portal](https://support.astronomer.io).
```

Feels a little too robotic to say "as Astronomer Support processes your request". I also added the word `also` given that you can check the status of a request by opening up your email thread, but _also_ you can check it in the support portal?

Author: @paolaperaza the current text is accurate. Emails are not always "replies". Whenever a ticket is modified or information is added, users receive an email notification. I tested it - the support team in India responded quite quickly! : )

Tone is subject to interpretation by every reader. What is robotic to you might be conversational to other readers. The use of contractions (you'll) indicates that this text is conversational and "conversational, friendly, and respectful without being overly colloquial or frivolous". The Google Developer Documentation Style Guide reminds writers that they should "provide information to someone who's looking for it and may be in a hurry". So, sentence length and structure need to as brief as possible. The longer a sentence and the more information packed it, the more difficult it is to read quickly. 

I have updated the text to: _You'll receive an email when your ticket is created and follow-up emails as Astronomer Support replies to your request. To check the status of a support request, you can also sign in to the Astronomer Support Portal_

## Submit a Support Request on the Astronomer Support Portal

Astronomer recommends that you submit support requests in the Cloud UI. If you can't access the Cloud UI, you can submit support requests on the [Astronomer Support Portal](https://support.astronomer.io).

> If you're new to Astronomer, you'll need to create an account on the Astronomer Support Portal to submit a support request. If you're working with a team and want to view support tickets created by other team members, use your work email or the domain you share with your team members for your account. If your team uses more than one email domain (for example, `@astronomer.io`), contact Astronomer and ask to have the additional domains added to your organization.

Reviewer 4: ```suggestion
If you're new to Astronomer, you'll need to create an account on the Astronomer Support Portal to submit a support request. We recommend that you create an account with the same email address you use to log in to Astro. If you're working with a team and want to view support tickets created by other team members, make sure to use an email address with the domain you share with other team members for your account (for example, `@astronomer.io`). If your team uses more than one email domain, contact Astronomer and ask to have the additional domains added to your organization.
```

List example in first instance of `domain`?

Author: @paolaperaza Here is how the text now appears:

_If you're new to Astronomer, you'll need to create an account on the Astronomer Support Portal to submit a support request. Astronomer recommends that you use the same email address that you use to access Astro for your account. If you're working with a team and want to view support tickets created by other team members, use your work email or the domain you share with other team members for your account (for example, `@astronomer.io`). If your team uses more than one email domain (for example, `@astronomer.io`), contact Astronomer and ask to have the additional domains added to your organization._

I try and avoid the use of _we_.

If you're an existing customer, sign in to the [Astronomer Support Portal](https://support.astronomer.io) and create a new support request.

## Monitor Existing Support Requests

If you've submitted your support request on the Astronomer Support Portal, sign in to the [Astronomer Support Portal](https://support.astronomer.io) to:

> - Review and comment on requests from your team.
> - Monitor the status of your requests.
> - Communicate with Astronomer Support.

Reviewer 4: ```suggestion
- Review and comment on other requests from your team.
- Monitor the status of all requests in your organization.
```

:::tip

> To add a teammate to an existing support request, cc them in a followup message within the email thread that was automatically generated when the ticket was created.

Reviewer 5: ```suggestion
To add a teammate to an existing support request, cc them when replying on the support ticket email thread.
```
